docroot/themes/custom/twentyeighteen/sass/base/_base.scss

```diff
@@ -35,80 +37,74 @@ body {
     }
   }
 }
+
+hr {
+  color: $teal;
+  border-width: 2pt;
+  border-style: solid;
+  margin: 2em 0;
+}
+
 // Base Headings
 //
 // Markup: headlines.twig
 //
 // Style guide: base.headlines
-h1 {
-  color: $sienna;
-  font-weight: 700;
-  font-family: $heading_font;
-}
-
+h1,
 h2,
 h3,
 h4,
 h5,
 h6 {
-  color: $heading;
+  font-family: 'Montserrat', Helvetica, Arial, sans-serif;
+  text-rendering: optimizeLegibility;
   font-weight: 700;
-  font-family: $heading_font;
-  padding: 0 0 0.3em;
-  margin: 0;
+  color: $light-teal;
 }
 
 h1 {
+  color: $sienna;
   font-size: 2em;
-  line-height: 165%;
 }
 
 h2 {
+  color: $teal;
```

Comment:
To make the code more condensed, let's remove the color attribute from each single declaration and into the above declaration where h1 - h6 are defined together. We can leave the h1 as is, and that will override it that one time.